### ISSUE TITLE
Username disappears if user.name == null

### ISSUE THREAD

**Issue by floofylily** [ChaoticLeah/pronoun-plugin#1]
Tested version: V3.3.7

When attempting to load the pronouns from a profile, the plugin will replace the display name of the user with user.name + pronouns
However, if someone doesn't have a custom display name set, user.name will be null, and so the name will appear something like this:
![image](https://github.com/ChaoticLeah/pronoun-plugin/assets/173606970/3f520728-0f2a-4885-88d0-5216a5c3e79c)
![image](https://github.com/ChaoticLeah/pronoun-plugin/assets/173606970/4eaf96db-d5f0-4fb3-82a2-8e71af17a894)

This seems to be done intentionally
```
if note.user.name == null {
    note.user.name = ""
}
```

However, I think that it would be better if instead of hiding the name, it was instead set to be equivalent to the user.username variable.

**Comment by floofylily** [comment]
On closer inspection, I think the issue is with this code on line 321:
```
var name = note.user.name
if name == null {
	name = note.user.username
}
```

It checks if note.user.name is null, but that condition will always fail since note.user.name is being set to "" if it is null.

**Comment by ChaoticLeah** [comment]
Ah right this may be from before the refactor.... oops. Yeah ill take a look into this. Do you have any profiles/posts where I can see this happen?

**Comment by ChaoticLeah** [comment]
fixed in V3.3.8
